cli/command/image/build/context.go

```diff
@@ -219,6 +224,8 @@ func getDockerfileRelPath(givenContextDir, givenDockerfile string) (absContextDi
 				absDockerfile = altPath
 			}
 		}
+	} else if absDockerfile == "-" {
```

Comment:
would this ever be true? I'm wondering if we would see the random name instead. And if so, then I think we can remove the "if" on line 243 below (`if givenDockerfile != "-" {`) since it would always be true.